Riot/Modules/Room/CellData/RoomBubbleCellData.m

```diff
@@ -1009,4 +1070,63 @@ - (NSString*)accessibilityLabelForAttachmentType:(MXKAttachmentType)attachmentTy
     return accessibilityLabel;
 }
 
+#pragma mark - URL Previews
+
+- (void)loadURLPreview
+{
+    // Get the last bubble component as that contains the link.
+    MXKRoomBubbleComponent *lastComponent = bubbleComponents.lastObject;
```

Comment:
Should this be wrapped in an `@syncronized`?

Reply:
🤷  Other folk might have a bettwe idea on the context of this area of code. Hopefully we won't have to ask these types of questions when in ViewModel/SwiftUI land. :)